lib/community/digest2/index.js

```diff
@@ -40,7 +40,13 @@ export const prepareDigestData = (id, type, opts = {}) => {
 
 const sendToUser = (user, data, opts = {}) =>
   personalizeData(user, data, opts).then(data =>
-    opts.dryRun || Email.sendSimpleEmail(user.get('email'), templateId, data))
+    opts.dryRun ||
+    Email.sendSimpleEmail(user.get('email'), templateId, data, {
+      sender: {
+        name: data.community_name || 'Hylo',
+        address: process.env.EMAIL_SENDER
```

Comment:
i don't think repeating `address` with the default value is necessary here, because `sendSimpleEmail` uses `lodash.merge`